README.md

```diff
@@ -601,6 +611,82 @@ To receive Alertmanager alerts, add webhook config like below to the `receivers`
        - "url": "http://notification-manager-svc.kubesphere-monitoring-system.svc:19093/api/v2/alerts"
 ```
 
+## Update
+
+There has a very big change since v1.0.0, all config crds are aggregated into a crd named `Config`, 
+and all receiver crds are aggregated into a crd named `Receiver`, and the crds now is the cluster scope crd.
+All the changes are as follows
+
+- All config crds are aggregated into a crd named `Config`.
+- All receivers crds are aggregated into a crd named `Receiver`.
+- Now the `Config`, `Receiver`, and `NotificationManager` are cluster scope crd.
+- The `NotificationManager` crd add a properties named `defaultSecretNamespace` which defines the default namespace to which notification manager secrets belong.
+- Now the namespace of the secret can be specified in `SecretKeySelector` like this. 
+  If the `namespace` of `SecretKeySelector` has be set, notification manager will get the secret in this namespace, 
+  else, notification manager will get the secret in the `defaultSecretNamespace`,
+  if the `defaultSecretNamespace` does not set, notification manager will get the secret in the namespace which the `notification manager operator` be in.
+
+```yaml
+    kind: Config
+    metadata:
+      labels:
+        type: tenant
+        namespace: default
+      name: user1-email-config
+    spec:
+      email:
+        authPassword:
+          key: password
+          name: user1-email-secret
+          namespace: kubesphere-monitoringsystem
+```
+
+- Move the configuration of DingTalk chatbot from dingtalk config to dingtalk receiver.
+- Move the chatid of DingTalk conversation from dingtalk config to dingtalk receiver.
+- Now the `chatid` of DingTalk conversation is an array types, and renamed to `chatids`.
+- Now the `port` of email `smartHost` is an integer type.
+- Now the `channel` fo slack is an array types, and renamed to `channels`.
+- Move the configuration of webhook from webhook config to webhook receiver.
+- Now the `toUser`, `toParty`, `toTag` of wechat receiver are array type.
+
```

Comment:
### Steps to migrate crds from v0.x to v1.0
You can update the v0.x to the latest version by following this:
- xx
- xx
- xx